
*****************************************************************************
Jun 28,2016 15:01:50 [info]   Execution Started of Test Class com.primeresponse.testcases.Enterprise.Verify_InternalFeedsMentionedAccounts and Test method testVerify_InternalFeedsMentionedAccounts
*****************************************************************************
Jun 28,2016 15:02:04 [info]  open application giturl
Jun 28,2016 15:03:12 [info]  log-in into application
Jun 28,2016 15:03:13 [info]  Select Selenium Test Ford Account if not selected
Jun 28,2016 15:03:48 [info]  Go to Enterprise > Internal Feed
Jun 28,2016 15:04:19 [info]  Go to Enterprise > Internal Feed
Jun 28,2016 15:04:49 [info]  success message
Jun 28,2016 15:05:00 [info]  Go to Track > Internal Feed
Jun 28,2016 15:10:11 [info]  Jun 28,2016 15:10:11 [info]  <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<< Error message >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
java.lang.AssertionError: Element Locator ://tbody[@id='internal-feed-items']/tr[4]/td[2]/div[1]/u Not found expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.primeresponse.util.DriverHelper.getText(DriverHelper.java:305)
	at com.primeresponse.pagehelper.EnterpriseHelper.matchInternalReviewDetails(EnterpriseHelper.java:756)
	at com.primeresponse.testcases.Enterprise.Verify_InternalFeedsMentionedAccounts.testVerify_InternalFeedsMentionedAccounts(Verify_InternalFeedsMentionedAccounts.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Jun 28,2016 15:10:11 [info]  End Execution of Test Class com.primeresponse.testcases.Enterprise.Verify_InternalFeedsMentionedAccounts
*****************************************************************************
